src/components/DuosHeader.jsx

```diff
@@ -82,7 +82,7 @@ export const headerTabsConfig = [
     link: '/admin_manage_dar_collections',
     children: [
       { label: 'DAR Requests', link: '/admin_manage_dar_collections' },
-      { label: 'Dataset Catalog', link: '/dataset_catalog' },
+      { label: 'Data Library', link: '/datalibrary', search: 'datalibrary' },
```

Comment:
What does this `search` field represent?

Reply:
The `search` value is used in a couple places in this file. I don't remember exactly how it works, but I'm pretty sure it has to do with displaying the correct tab based on your role and where you have navigated to/from. For example, if you have multiple roles, each one with access to the same sub-tab (i.e. data library), then this feature helps restore the correct sub-tab when the user navigates between higher level role tabs.